src/main/scala/io/moia/protos/teleproto/MigrationImpl.scala

```diff
@@ -139,8 +139,8 @@ class MigrationImpl(val c: blackbox.Context) extends FormatImpl {
     // collect the expressions for the constructor of the target proto
     val passedExpressions =
       paramMigrations.map {
-        case Automatically(tree: Tree, _) => tree
```

Comment:
The type of `tree` is always `Tree` but the pattern match is marked as non-exhaustive with the annotation. So I just removed it.